### PR TITLE
Binary checksums

### PR DESCRIPTION
**Pull request by CarlBeek**
In order to offer signatures and checksums, this PR makes python builds deterministic and adds sha256 hashes to circle ci.

### CODE REVIEW

**Comment by hwwhww** [comment]
There is an error message in
https://app.circleci.com/pipelines/github/ethereum/eth2.0-deposit-cli/330/workflows/2ac433fd-43dd-4a21-946e-dd7b49bd983b/jobs/2669

```
certUtil -hashfile $ZIP_FILE_NAME > \tmp\artifacts\checksum.txt SHA256
Write-Progress : Win32 internal error "Access is denied" 0x5 occurred while reading the console output buffer. Contact 
Microsoft Customer Support Services.
At 
C:\Windows\system32\WindowsPowerShell\v1.0\Modules\Microsoft.PowerShell.Archive\Microsoft.PowerShell.Archive.psm1:1132 
char:9
+         Write-Progress -Activity $cmdletName -Status $status -Percent ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ReadError: (:) [Write-Progress], HostException
    + FullyQualifiedErrorId : ReadConsoleOutput,Microsoft.PowerShell.Commands.WriteProgressCommand
 


    Directory: C:\tmp


Mode                LastWriteTime         Length Name                                                                  
----                -------------         ------ ----                                                                  
d-----       10/14/2020   8:03 PM                artifacts                                                             

CircleCI received exit code 0
```

but the artifacts output is correct.

File: .circleci/config.yml [hunk 1]
```diff
@@ -117,6 +117,7 @@ jobs:
       - run:
           name: Build with build.spec
           command: |
+            export PYTHONHASHSEED=42
```
[the remark this answers is not in the document]
Comment (CarlBeek) [reply .circleci/config.yml:120]: It is a fixed/magic number. 42, pi, bitcoin genesis hash, and the  HD DVD AACS encryption key are my gotos.

**Review by hwwhww — APPROVED** [review]
LGTM

**Comment by CarlBeek** [comment]
> There is an error message in
> https://app.circleci.com/pipelines/github/ethereum/eth2.0-deposit-cli/330/workflows/2ac433fd-43dd-4a21-946e-dd7b49bd983b/jobs/2669
> 
> but the artifacts output is correct. I think it's fine.

Agreed output remains correct. It appears to be a common problem when running powershell remotely. In our case, what is failing is the progress bar cannot print to stdout

**Pull request merged**